ads/inabox/position-observer.js

```diff
@@ -14,14 +14,18 @@
  * limitations under the License.
  */
 
-import {layoutRectLtwh, LayoutRectDef} from '../../src/layout-rect';
+import {
+  layoutRectLtwh,
+  LayoutRectDef,
+  layoutRectFromDomRect,
+} from '../../src/layout-rect';
 import {Observable} from '../../src/observable';
 import {throttle} from '../../src/utils/rate-limit';
 
 /**
  * @typedef {{
- *   viewport: !LayoutRectDef,
- *   target: !LayoutRectDef
+ *   viewportRect: !LayoutRectDef,
+ *   positionRect: !LayoutRectDef,
```

Comment:
targetRect? 
viewportRect is "the rect of the viewport"
feel weird to say: positionRect is "the rect of the position"